src/python/foglamp/translators/omf_translator.py

```diff
@@ -45,52 +39,44 @@
 _message_list = {
 
     # Information messages
-    "i000001": "operation successfully completed",
-    "i000002": _module_name + " - Started",
+    "i000001": " ",
```

Comment:
yes, it is no more used, I avoided to pack/change all the used id for now (I will do)  but at least I have highlighted.